### PR TITLE
output folder update

### PR DESCRIPTION
**Pull request by saggir**
The output folder will be the current opened workspace root path.
If there is no opened workspace, it will be generated under home/projects

### CODE REVIEW

File: backend/src/yeomanui.ts [hunk 1]
```diff
@@ -82,6 +84,9 @@ export class YeomanUI {
     this.currentQuestions = {};
     this.setGenFilter(genFilter);
     this.customQuestionEventHandlers = new Map();
+    if (vscode.workspace && vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length > 0) {
+      YeomanUI.cwd = vscode.workspace.workspaceFolders[0].uri.path;
+    }
```
Comment (slavik-lvovsky) [inline backend/src/yeomanui.ts]: it is better to pass first workspace folder object or even it's uri.path to the yeomanui constructor.
We do not need dependency to vscode here.

When you do it please use lodash

Comment (tomer-epstein) [reply backend/src/yeomanui.ts]: 1. pass path to the constructor.
2. why only workspaceFolders[0] ? is this by design (if it's multi-root worksapce the take the 'root ws' folder)?

Comment (saggir) [reply backend/src/yeomanui.ts]: workspaceFolders[0] holds the current active workspace, do u see a use case that we need another workspace?

File: backend/src/extension.ts [hunk 2]
```diff
@@ -147,8 +147,8 @@ export class YeomanUIPanel {
 		this.rpc = new RpcExtension(this.panel.webview);
 		const outputChannel: YouiLog = new OutputChannelLog();
 		const vscodeYouiEvents: YouiEvents = new VSCodeYouiEvents(this.rpc, this.panel);
-
-		this.yeomanui = new YeomanUI(this.rpc, vscodeYouiEvents, outputChannel, this.logger, YeomanUIPanel.genFilter);
+		const outputFolder = _.get(vscode, "vscode.workspace.workspaceFolders[0].uri.path");
```
Comment (slavik-lvovsky) [inline backend/src/extension.ts]: remove vscode before workspace (in the string)

Comment (saggir) [reply backend/src/extension.ts]: fixed

File: backend/src/yeomanui.ts (hunk 1 above)
Comment (tomer-epstein) [reply backend/src/yeomanui.ts]: sounds good to me.

File: backend/src/yeomanui.ts [hunk 3]
```diff
@@ -414,4 +416,17 @@ export class YeomanUI {
...
   }
+
+  public setCwd(cwd: string){
+    if(cwd){
+      YeomanUI.cwd=cwd;
+    }
+    else{
+      YeomanUI.cwd=path.join(os.homedir(), 'projects');
+    }
+  }
```
Comment (tomer-epstein) [inline backend/src/yeomanui.ts]: no need foe else!
https://github.com/SAP/yeoman-ui/pull/192/files#diff-d929b362927bd7b256541b9d8b48f0daR45

**Pull request merged**